openmetadata-ui/src/main/resources/ui/src/components/common/title-breadcrumb/title-breadcrumb.component.tsx

```diff
@@ -113,7 +113,7 @@ const TitleBreadcrumb: FunctionComponent<TitleBreadcrumbProps> = ({
               <li
                 className="d-flex items-center breadcrumb-item"
                 data-testid="breadcrumb-link"
-                key={link.name}>
+                key={link.name + index}>
```

Comment:
if the `index` here is array index please don't use that in a `key`. This will throw lint warning.

Reply:
@Ashish8689 I don't see the changes to remove the `index` from the key prop. Can you please check?

Reply:
@aniketkatkar97 We have wrapped it in template literals along with the name, so it won't be throwing an error.